src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -186,10 +181,9 @@ public Response deleteDaaForUser(
       @PathParam("daaId") Integer daaId,
       @PathParam("userId") Integer userId) {
     try {
-      User user = userService.findUserByEmail(authUser.getEmail());
-      List<LibraryCard> libraryCards = libraryCardService.findLibraryCardsByUserId(userId);
```

Comment:
This is a bug. When we remove DAAs from a library card, we need to remove them from the user passed into the method via `userId`. This code will remove a DAA from the AuthUser's library card.

Reply:
Good catch!

Reply:
This is currently not enabled in any environment so I want to tackle any explicit functional changes to DAA work as a distinct task. We will need to re-evaluate the entire feature before we do any further work here.

Reply:
Made a ticket to remind ourselves: https://broadworkbench.atlassian.net/browse/DT-1704